asym_crypto_yaml/crypto.py

```diff
@@ -0,0 +1,148 @@
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import serialization
+from cryptography.hazmat.primitives.asymmetric import padding, rsa
+from cryptography.hazmat.primitives import hashes
+import yaml
+import base64
+
+# See https://cryptography.io/en/latest/
+
+def encode(s):
```

Comment:
It would be good to have docstrings on functions.  This one takes "s", which sounds like a string, but then uses it with encodebytes, which expects bytes?